Project.toml

```diff
@@ -10,11 +10,11 @@ YAML = "ddb6d928-2868-570f-bddf-ab3f9cf99eb6"
 goaws_jll = "a437b02d-256c-57cc-b5c6-817ed6633e35"
 
 [compat]
-AWS = "1.8 - 1.90"
+AWS = "1.91"
```

Comment:
I'll look into supporting older versions of AWS.jl as this would allow for better compatiblity

Reply:
I think we intended on using `1.80` here as `1.8.0` doesn't support `AbstractAWSConfig` which this package requires. We at a minimum require `1.63` which added support for `@service`

Reply:
Restricting further to AWS.jl 1.78 as this version allows us to use Compat.jl 4 which is needed to run the tests. I'd rather not state this package supports AWS.jl 1.63 if we can't actually tests against it